packages/wrangler/src/deploy/deploy.ts

```diff
@@ -757,6 +758,10 @@ See https://developers.cloudflare.com/workers/platform/compatibility-dates for m
 		let workerBundle: FormData;
 
 		if (props.dryRun) {
+			if (config.containers) {
+				await buildContainers(config, workerTag ?? "worker-tag");
```

Comment:
Should this pass in `true` for `dryRun`?

Reply:
I dont think we accept an optional parameter for buildContainers?